### ISSUE TITLE
weird windows edge

### ISSUE THREAD

**Issue by TRINCETT** [mydockfinder/mydockfinder#549]
Since the last update I have been having an error where a space is left between the windows
And it usually happens when an app starts maximized.
For example telegram Spotify (Both when are opened from the System Tray) or the settings app with the Win+I shortcut.

![image](https://github.com/mydockfinder/mydockfinder-for-Win10-Win11/assets/93570862/8e799b70-240f-46de-b8ba-64a00cd53dc5)

![image](https://github.com/mydockfinder/mydockfinder-for-Win10-Win11/assets/93570862/0bf95866-1fc0-475d-85fd-38c724fcc011)

**Comment by mydockfinder** [comment]
应该是窗口居中的问题，下个版本修复，可以先把窗口居中功能关闭
It should be a problem with the center of the window, and the next version will fix it, you can first turn off the window center function




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年7月21日(星期五) 晚上11:29
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; 
主题: [mydockfinder/mydockfinder-for-Win10-Win11] weird windows edge (Issue #549)






自从上次更新以来，我一直有一个错误，在窗口之间留下一个空间
它通常发生在一个应用程序启动最大化。
例如电报Spotify（两者当从系统托盘打开）或设置应用程序与Win+I快捷方式。





—
直接回复此邮件，在GitHub上查看，或取消订阅.
***@***.***与&gt;.

**Comment by TRINCETT** [comment]
Nope, as you can see here i have that option disabled because i reinstall the program and it keeps happening.

https://github.com/mydockfinder/mydockfinder-for-Win10-Win11/assets/93570862/c6591e4c-0ee0-4d12-86b6-cb754e14defa

**Comment by mydockfinder** [comment]
Because your window is not maximized, just plain to spread the screen. If it is maximized, there is no problem
&nbsp;


№じ&nbsp;尐乄鈊~→☆
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年8月6日(星期天) 凌晨4:00
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] weird windows edge (Issue #549)





 
Nope, as you can see here i have that option disabled because i reinstall the program and it keeps happening.
 
https://github.com/mydockfinder/mydockfinder-for-Win10-Win11/assets/93570862/c6591e4c-0ee0-4d12-86b6-cb754e14defa
